lib/manager/npm/extract/yarn.ts

```diff
@@ -41,3 +46,29 @@ export async function getYarnLock(
     return { isYarn1: true, cacheVersion: NaN, lockedVersions: {} };
   }
 }
+
+export async function getYarnRc(
+  packageFilePath: string,
+  config: ExtractConfig
+): Promise<RcFile> {
+  if (is.string(config.yarnrc) || config.localDir === undefined) {
+    return undefined;
+  }
+  const yarnRcFileNames = [
+    '.yarnrc',
+    '.yarnrc.yml',
```

Comment:
`.yarnrc.yml` overrides (takes precedence over) `.yarnrc` in Yarn 1 (https://git.io/JUcco, https://github.com/renovatebot/renovate/pull/7220#discussion_r485383364).